modules/styling/lib/cs.ts

```diff
@@ -260,6 +260,26 @@ export function csToProps(input: CSToPropsInput): CsToPropsReturn {
   // At this point, `input` is either an array or a `Record<T, string>`. If it isn't an array, it
   // must be a style object for setting CSS variables. So set the `style` attribute
   if (!Array.isArray(input)) {
+    // The input is an object. We'll separate variables from static styles. If static styles are
+    // detected here, a dynamic mode should be detected and done for development. The transform
+    // should take care of extracting static styles out of a render function and this function
+    // should not be run.
+    let hasStaticStyles = false;
+    const staticStyles: CSSObject = {};
+    const cssVars: Record<string, string> = {};
+    for (const key in input) {
```

Comment:
Isn't safer to check if key starts with '--' for css vars instead of just '-'? will something like `-webkit-transform` be recognized as css var? I don't know if there is any possibility of passing this though

Reply:
Yes. Good catch

Reply:
I think it is unlikely that someone is using a vendor prefix anymore, but a vendor prefix is still a valid style key. I've updated the check to look for `--`